### PR TITLE
CU : Add channels

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Astra streams, controlled by the ICRT.

### CODE REVIEW

File: channels/cu.m3u [hunk 1]
```diff
@@ -1,3 +1,13 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="CubaVision.cu" tvg-name="CubaVision" tvg-country="CU" tvg-language="" tvg-logo="" group-title="",CubaVision
-http://190.122.96.187:8888/http/010
```
Comment (freearhey) [inline channels/cu.m3u:3]: I'm sorry, but I don't understand why you deleted this link?

Comment (LaneSh4d0w) [reply channels/cu.m3u:3]: Not working.

Comment (freearhey) [reply channels/cu.m3u:3]: Well, it seems to be working now.
image

Comment (LaneSh4d0w) [reply channels/cu.m3u:3]: Reintegrating it, changing its name since it's the International version.

**Review by freearhey — APPROVED** [review]

**Pull request merged**